src/tracing/types.rs

```diff
@@ -156,6 +156,17 @@ pub struct DecodedCallLog {
     pub params: Option<Vec<(String, String)>>,
 }
 
+/// portable representation of call Log Styling
+#[derive(Debug, Default, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum LogStyle {
```

Comment:
this is fine for now, but I think we might want to give more flexibility here by keeping  some `anstyle` type here

we'd need to figure out serde for it, so can be done in a follow up